src/envs/multi-armed-bandit.jl

```diff
@@ -0,0 +1,48 @@
+module MultiArmedBanditEnv
+
+using ..Reinforce
+using Distributions
+
+export MultiArmedBandit
+
+"""
+The multi-armed bandit environment.
+
+    MultiArmedBandit(k, n = 100; σ = 1)
+
+- `k` is the number of available arms.
+  The reward distribution for each arms is a Normal distribution centered
+  at the range of [-1, 1] and the standard deviation is 1.
```

Comment:
possible edit: "The reward distribution for each ~arms~ arm is a Normal distribution with its mean drawn uniformly from the range [-1, 1]..."